projects/output-and-error-handling/client/main.go

```diff
@@ -0,0 +1,182 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"net/http"
+	"os"
+	"strconv"
+	"time"
+)
+
+// Comments ----------
+// I write a lot of comments, it has helped me the entire course to learn, understand and remember
+// They can all be removed at the end to clean it up
+
+// Error Messages ----------
+// The Errors are deliberately numbered [x] so I could follow them up the call stack
+// They can be replaced with "user friendly" messages later
+
+func parseResponseBody(response *http.Response) (string, error) {
```

Comment:
I'm not sure I'd use "parse" in this function name - this function isn't really _parsing_ the value (taking it in one format an interpreting it in some domain), it's really just reading it (and converting it to a string)?